circle.yml

```diff
@@ -1,26 +1,37 @@
 machine:
```

Comment:
Since you are touching this file can you use https://github.com/codacy/codacy-cppcheck/blob/master/.circleci/config.yml ?